src/Manifolds.jl

```diff
@@ -33,6 +33,7 @@ import Base:
     transpose,
     zero
 import Distributions: _rand!, support
+import LightGraphs: AbstractSimpleGraph, is_directed
```

Comment:
These could probably be imported with `using` instead? Similarly below in `import SimpleWeightedGraphs: AbstractSimpleWeightedGraph, get_weight`.